libraries/testing/tester.cpp

```diff
@@ -413,14 +413,14 @@ namespace eosio::testing {
                trace->except->dynamic_rethrow_exception();
             }
             itr = unapplied_transactions.erase( itr );
-            res.traces.emplace_back( std::move(trace) );
+            res.unapplied_transaction_traces.emplace_back( std::move(trace) );
          }
 
          vector<transaction_id_type> scheduled_trxs;
          while ((scheduled_trxs = get_scheduled_transactions()).size() > 0 ) {
             for( const auto& trx : scheduled_trxs ) {
                auto trace = control->push_scheduled_transaction( trx, fc::time_point::maximum(), fc::microseconds::maximum(), DEFAULT_BILLED_CPU_TIME_US, true );
-               res.traces.emplace_back( trace );
+               res.unapplied_transaction_traces.emplace_back( trace );
```

Comment:
Remove this line. These are NOT unapplied transactions.

Reply:
For sure.